app/src/main/java/com/demoapp/database/DataDuo.kt

```diff
@@ -6,24 +6,23 @@ import androidx.room.Query
 import com.demoapp.model.CategoryEntity
 import com.demoapp.model.DataEntity
 
-
 @Dao
 interface DataDuo {
     @Query("SELECT  * FROM category_entity")
-    fun getAllCategories(): List<CategoryEntity>
+    suspend fun getAllCategories(): List<CategoryEntity>
 
     @Insert
-    fun insertCategories(categoryEntity: CategoryEntity)
+    suspend fun insertCategories(categoryEntity: CategoryEntity)
 
     @Query("DELETE FROM category_entity")
-    fun removeCategoryData()
+    suspend fun removeCategoryData()
 
     @Query("SELECT  * FROM data_entity")
-    fun getData(): List<DataEntity>
+    suspend fun getData(): List<DataEntity>
 
     @Insert
-    fun insertData(dataEntity: DataEntity)
+    suspend fun insertData(dataEntity: DataEntity)
 
     @Query("DELETE FROM data_entity")
-    fun removeData()
+    suspend fun removeData()
 }
```

Comment:
end line. Change this in your Android Studio settings to ensure end line in all files.